timeseries/src/autogluon/timeseries/models/chronos/model.py

```diff
@@ -366,7 +621,7 @@ def _predict(
         return TimeSeriesDataFrame(df)
 
     def _more_tags(self) -> Dict:
-        return {"allow_nan": True}
+        return {"allow_nan": True, "can_use_val_data": True}
```

Comment:
```suggestion
        return {"allow_nan": True, "can_use_val_data": self._get_model_params()["fine_tune"]}
```
This will allow us to save some time on preprocessing val_data if fine-tuning is disabled